test/Npgsql.NativeAotTests/Npgsql.NativeAotTests.csproj

```diff
@@ -2,8 +2,6 @@
   <PropertyGroup>
     <OutputType>exe</OutputType>
     <PublishAot>true</PublishAot>
-    <!-- We need to override TargetFrameworks here to avoid build failures  -->
```

Comment:
This seems pretty obscure - is it worth leaving here for the future?

Reply:
Leaving what exactly? That line is removed.

Reply:
Ah, you mean to restore that? Not really that was just due to us referencing both netcoreapp3.1 and .net 8 for test.